crates/sui-rpc-api/proto/sui.node.v2.proto

```diff
@@ -0,0 +1,352 @@
+syntax = "proto3";
+
+package sui.node.v2;
```

Comment:
Should we add a version number to the package name i.e. sui.node.v2.1 for supporting future backward incompatible changes without breaking existing clients. For example - adding a new request and response type with an endpoint will not break existing clients and could go under `sui.node.v2.2"

Reply:
That's what the `v2` is supposed to convey already, unless i'm misunderstanding.